packages/core/src/components/sections/CrossSellingShelf/OverriddenDefaultCrossSellingShelf.ts

```diff
@@ -0,0 +1,11 @@
+import { getOverriddenSection } from 'src/sdk/overrides/getOverriddenSection'
+import type { SectionOverrideDefinition } from 'src/typings/overridesDefinition'
+
+/**
+ * This component exists to support overrides 1.0
+ *
+ * This allows users to override the default CrossSellingShelf section present in the Headless CMS
+ */
+export const OverriddenDefaultCrossSellingShelf = getOverriddenSection({
+  section: 'CrossSellingShelf',
+} as SectionOverrideDefinition<'CrossSellingShelf'>)
```

Comment:
Is this type conversion needed?

Reply:
No, it isn't. Removed [here](https://github.com/vtex/faststore/pull/2168/commits/ef46c9fbffa5af0f10b261edf2088634ced8c9c5).